master/buildbot/util/logger.py

```diff
@@ -35,5 +35,10 @@ def _log(self, format, *args, **kwargs):
         critical = _log
 
         def failure(self, format, failure, *args, **kwargs):
-            log.error(failure, format.format(args, **kwargs))
```

Comment:
Updated in 1e9a5432a6a9ac314186830c138a487e8ab0fdc3. This should solve the AttributeError I was getting initially.